plugins/memblaze/memblaze-nvme.c

```diff
@@ -220,8 +225,8 @@ static void show_memblaze_smart_log_new(struct nvme_memblaze_smart_log *s,
     printf("%-32s: %3d%%       %s%u%s%u%s%u\n", STR17_01,  *nm, STR17_03, *raw,
         STR17_04, *(raw+2), STR17_05, *(raw+4));
     /* 18 RAISIN_SI_VD_POWER_LOSS_PROTECTION */
-    get_memblaze_new_smart_info(smart, RAISIN_SI_VD_POWER_LOSS_PROTECTION, nm, raw);
-    printf("%-32s: %3d%%       %"PRIu64"\n", STR18_01, *nm, int48_to_long(raw));
+    // get_memblaze_new_smart_info(smart, RAISIN_SI_VD_POWER_LOSS_PROTECTION, nm, raw);
+    // printf("%-32s: %3d%%       %"PRIu64"\n", STR18_01, *nm, int48_to_long(raw));
```

Comment:
I suggest you remove the code instead just commenting it out. 

Anyway, looking through the memblaze-nvme.c plugin I noticed that is breaking coding style rules in a lot of places.
The result is that the code is harder to read and to maintain IMO. Would you mind to cleanup the file?

Reply:
OK, I will delete these useless code.
For the coding style, there are much work to cleanup. We will create a new pr for that.

Reply:
Excellent, sounds like a plan! Let's do this.